test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs

```diff
@@ -200,6 +200,43 @@ public WebHostFunctionalTests(ITestOutputHelper output) : base(output)
             }
         }
 
+        [Fact]
+        public void LoggingConfigurationSectionPassedToLoggerByDefault()
+        {
+            try
+            {
+                File.WriteAllText("appsettings.json", @"
+{
+    ""Logging"": {
+        ""LogLevel"": {
+            ""Default"": ""Warning""
+        }
+    }
+}
+");
+                using (var webHost = WebHost.Start(context => context.Response.WriteAsync("Hello, World!")))
+                {
+                    var factory = (ILoggerFactory)webHost.Services.GetService(typeof(ILoggerFactory));
+                    var logger = factory.CreateLogger("Test");
+
+                    logger.Log(LogLevel.Information, 0, "Message", null, (s, e) =>
+                    {
+                        Assert.True(false);
+                        return string.Empty;
+                    });
+
+                    logger.Log(LogLevel.Warning, 0, "Message", null, (s, e) =>
+                    {
+                        return string.Empty;
```

Comment:
How can we tell that this log *did* get written out? Can either use mocks, or at least set a flag to `true` or something and then assert that.